bt_project/auction_auction[] auctions
int64 auction_ID
